pyproject.toml

```diff
@@ -17,39 +17,33 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-# Required for Python 3.7 for now. See: https://stackoverflow.com/a/73932581/194311
-importlib-metadata = "4.13.0"
-# Used for local development
-nautobot = { version = "^1.4.0", optional = true }
+nautobot = { version = ">=2.0.0-rc.1", optional = true }
```

Comment:
```suggestion
nautobot = { version = ">=2.0.0-rc.2", optional = true }
```

Reply:
Works for me with the current value `rc.1`. Could you pls. check there is no `invoke.yml` on your localhost or something similar?